### ISSUE TITLE
Request for Grade

### ISSUE THREAD

**Issue by lolasundara** [Brewskiez/brewskiez-app#32]
| | 0 points | 1 points | 2 points | 3 points |
| --- | --- | --- | --- | --- |
| **Accessibility** | No considerations | Added some alt attributes, nothing else | Just the basics: alt attributes, roles | Accessibility is well considered and tested |
| **Code quality** | Barely started | Indentation is barely existent, lots of validation errors, very poor semantics | Decent indentation, just a couple validation errors, decent semantics | Well indented, fully valid, good semantics |
| **Responsive** | Not Responsive, screen sizes not working | Things are tested however not everything has been tested using different sites, Chrome, Firefox etc  | Some screen sizes are responsive | Everything is responsive and tested |
| **Website Navigation** | Barely started, not linked | Decent amount of linked pages | Some pages are linked but pages are not linked to home page | All pages are linked, buttons are noticeable, links are connected to correct information |
| **Performance** | Website is slow, not loading | Some pages load where the others do not |  Still slow loading, but still usable | No loading issues |

Overall, we believe we should receive an A- on our final website. 

The reason being, we were able to design and create a responsive and creative pattern library that showcases our brand identity, Brewkiez, through its use of colours and design. Along with the pattern library, we were able to develop a responsive and accessible website. We feel that we have successfully used the knowledge learned throughout the past web development courses in order to create this website, meeting all elements needed such as the use of HTML, CSS, Java, as well as Jekyll, Markdown and YAML. 

Our Accessibility is well considered and tested due to the use of ARIA labels etc. The code is clean in which it can be easily altered and edited if needed for future code references. Our code includes a good use of semantics, along with good indentation allowing the website to function to its best ability. Our website was tested on all sites and responds to each screen size with the media queries and CSS code. We feel our code is up to standard, using our patterns to the advantage. Rather than coding everything individually, we used our pattern library snipets in order to create a simplified version of our overall code. All pages are linked to the navigation which allows our website to be easily navigated. All pages load without any lagging allowing users to view our website efficiently. 

A few problems that we encountered included merging code without overwriting the other teammates work. This was resolved by creating our own individual branches, and deleting them once they have been approved by another teammate. 

In conclusion, we believe we have implemented and used all code necessary in order to complete a responsive website that brands the companies identity. Each section has the capability to be changed and altered easily for future reference and usage.

@thomasjbradley

**Comment by cayangaannika** [comment]
Accessibility: 3
Code quality: 3
Responsiveness: 3
Website navigation: 2
Performance: 3
